models/reviews.model.js

```diff
@@ -66,19 +66,28 @@ return rows.map((review) => {
           return review;
```

Comment:
Rather than deleting properties from this object, id prefer you just select the things you actually do want in the sql query